json/three_tracks.json

```diff
@@ -0,0 +1,687 @@
+{
+  "name": "Crypto template",
+  "config": {
+    "default_track_locations": [{"name": "Lotte Lehman Hall"}, {"name": "Campbell Hall"}],
```

Comment:
out of curiosity, what happens for the third track if you only put in two location names? does it default to a blank or...?

Reply:
This is a good question, but I don't think we ever actually used the default locations. We may wish to simply remove them because I don't think they are exposed in the editor.